### ISSUE TITLE
Uncaught TypeError: Cannot assign to read only property 'prototype' of function Map()

### ISSUE THREAD

**Issue by huang47** [facebook/react-devtools#313]
repro steps

1. install react-devtools(built from latest master) on chrome (lastet stable)
2. open https://facebook.github.io/react/
3. open inspector and see following error and stuck at `Connecting to React...`

![screen shot 2016-01-16 at 3 42 16 pm](https://cloud.githubusercontent.com/assets/157390/12375099/be155a80-bc67-11e5-985e-28525979b64b.png)

**Comment by dckesler** [comment]
I'm also having this issue. Both with Map and Set. When I remove
```javascript
Map: 'window.__REACT_DEVTOOLS_GLOBAL_HOOK__.nativeMap',
Set: 'window.__REACT_DEVTOOLS_GLOBAL_HOOK__.nativeSet',
```
from `webpack.backend.js` it seems to fix it for now.
Both errors seems to be related to `immutable.js`.

**Comment by huang47** [comment]
@dckesler fixed in #317
